source/best-practices/backup.rst

```diff
@@ -1,61 +1,90 @@
 备份与还原
 ==========
 
-**备份！备份！备份！**
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（作者）
+:最近更新日期: 2021-01-30
+:预计阅读时间: 20 分钟
+
+.. warning::
+
+   本章正在编写中，尚未完成。
+
+----
+
+备份！备份！备份！
+-------------------
 
 尽管不经常发生，但电脑随时可能会坏掉或被盗，电脑硬盘也随时可能会出问题，
-也可能不小心使用一个命令把几个月的工作误删了。所以，备份非常重要，不进行必要的备份
-是十分危险的。
+也可能不小心使用一个命令（如 ``rm -r *``）把几个月的工作误删了。所以，备份非常重要，不进行必要的备份
+是十分危险的。笔者和一些同行多次体验过数据误删或硬盘损坏又未备份带来的痛苦。
 
-备份的目的
-----------
+如果\ **文件非常重要**\ ，丢失后\ **恢复难度极大**\ ，或\ **恢复时间极长**\ ，
+强烈推荐备份一下。例如，毕业论文、脚本、代码、无法再次获取的数据、经过长时间处理得到的资料。
 
-- 仅备份重要文件
-- 仅备份家目录
-- 全盘备份
+.. warning::
 
-备份的介质
-----------
+   日常科研工作中，电脑硬盘经常进行大量的读写操作，直接影响硬盘的寿命。我们的经验表明，电脑硬盘的平均
+   寿命大约为 5 年。当然，硬盘的品质和使用情况不同，其寿命也有所不同。
 
-- 硬盘备份
-- 网盘备份
+备份策略
+---------
 
-我们主要讨论硬盘备份。
+备份策略简单概括有两种：
+
+- 备份家目录或其部分子目录
+- 全盘备份
+
+不同备份策略的主要区别是所需的硬盘空间和备份时间，以及硬盘摔坏带来的损失。
+移动硬盘是最常见的备份设备，其特点是携带方便、价格便宜。近几年，移动硬盘发展迅速，
+市场上常见的存储空间大小有 4TB 和 5TB，单价不到 1000 块，基本可以满足日常科研的备份需求。
+一般而言，第一次备份需要较长时间，后续的备份往往只同步改动过的文件，所需时间较短。
+考虑以上两点因素以及硬盘摔坏带来的精神和身体上的损失，我们推荐备份家目录或者至少
+备份家目录下重要的子目录。有需求的用户可以考虑全盘备份。
 
 Linux
 ------
 
 rysnc
 ^^^^^^
 
-至少每周将工作电脑中的文件备份到移动硬盘中。推荐使用 rsync 进行备份，其用法为::
+至少每周将工作电脑中的文件备份到移动硬盘中。推荐使用 ``rsync`` 命令进行备份，其用法为::
 
-    rsync --delete -av /home/seisman/ /data1/seisman/
+    $ rsync --delete -av /home/seismo-learn/ /data1/seismo-learn/
 
-该命令作用是将 /home/seisman/ 目录完整同步到 /data1/seisman/ 目录下。
+该命令作用是将 /home/seismo-learn/ 目录完整同步到 /data1/seismo-learn/ 目录下。
```

Comment:
1. Change `/home/seismo-learn/` to <code>:file:`/home/seismo-learn`</code>
2. Are you sure that we need the trailing slash (i.e., `/home/seismo-learn/`, not `/home/seismo-learn`
3. If yes to 2, better to add a warning that the trailing slash is important
4. Change `/data1` to a more reasonable name `/mnt/xxx`?

Reply:
We will work on it in another PR, right? I just revise this part a little.

We better prepare a script`backup.sh`

Reply:
> 1. Change `/home/seismo-learn/` to `` :file:`/home/seismo-learn` ``
done

> 2. Are you sure that we need the trailing slash (i.e., `/home/seismo-learn/`, not `/home/seismo-learn`
> 3. If yes to 2, better to add a warning that the trailing slash is important

We will work on this part in another PR?

> 4. Change `/data1` to a more reasonable name `/mnt/xxx`?
done

Reply:
OK